### PR TITLE
issue number #68 additional enhancements

### PR DESCRIPTION
**Pull request by kaleem68**
Hey @Hopelezz, does this make sense to you? Also, @Jutanium, can you review this?

### CODE REVIEW

File: src/routes/guides/getting-started-with-solid/building-ui-with-components.mdx [hunk 1]
```diff
@@ -315,7 +315,7 @@ There are a couple terms worth learning when we talk about how components relate
 
 ### Passing props between components
 
-Components can communicate with their child components by passing properties, which are commonly called `props`. To demonstrate the power of `props`, let's customize our bookshelf by adding our name to it! In the example below, we update our app to display "Solid's Bookshelf," but you should use your own name.
+Components can communicate with their child components by passing properties called `props`. To demonstrate the power of `props`, let's customize the bookshelf by adding your name! In the example below, we update the app to display "Solid's Bookshelf," but you should use your name.
```
Comment (Hopelezz) [inline src/routes/guides/getting-started-with-solid/building-ui-with-components.mdx:318]: @Jutanium Do we have a standardized language we are using? 

Customize (American) vs Customise (British)

**Review by Hopelezz — CHANGES_REQUESTED** [review]
@kaleem68 Hey, everything looks good on my end. I just want to see if @Jutanium has any feedback.

File: src/routes/guides/getting-started-with-solid/building-ui-with-components.mdx (hunk 1 above)
Comment (crutchcorn) [reply src/routes/guides/getting-started-with-solid/building-ui-with-components.mdx:318]: > passing properties called `props`

Makes it feel like mentioning `props` is redundant. In reality, highlighting that they're also called `props` is a helpful distinction to newcomers.

Could we maybe try:

```suggestion
Components can communicate with their child components by passing properties, often shortened to `props`. To demonstrate the power of `props`, let's customize the bookshelf by adding your name! In the example below, we update the app to display "Solid's Bookshelf," but you should use your name.
```

Or some variation of such?

Comment (Hopelezz) [reply src/routes/guides/getting-started-with-solid/building-ui-with-components.mdx:318]: Ohhh, this sounds better! I like it.

Comment (Jutanium) [reply src/routes/guides/getting-started-with-solid/building-ui-with-components.mdx:318]: @Hopelezz we're using American English - just added to the writing guide!
@kaleem68 if you could review @crutchcorn's suggestion and hit "Commit suggestion," that would be awesome!

**Review by Jutanium — APPROVED** [review]
LGTM!
I'll make a future PR that adjusts the exclamation points in the tutorial.

**Comment by Hopelezz** [comment]
@Jutanium , we can make that "good first issue" tag and use them to guide users through the PR process.

**Pull request merged**